sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 5.2.0-beta.2 (Unreleased)
 
 ### Features Added
+- Added property `RecognizeCustomEntitiesAction` to `TextAnalyticsActions` to support the new 'Recognize Custom Entities' API. This action can be used to get custom entities for an input document or batch of documents.
```

Comment:
Not sure if this is the best description for the feature. If there are any suggestions I will be glad to modify the existing one.

Reply:
Since custom classification PR is merged. Might to pull the latest changes and see if rewording is neccessary. It looks good to me as well right now.